src/Files.Core/Utils/Cloud/CloudProviders.cs

```diff
@@ -1,27 +1,44 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-namespace Files.Shared.Cloud
+namespace Files.Core.Utils.Cloud
 {
 	public enum CloudProviders
 	{
 		OneDrive,
+
```

Comment:
We don't usually add empty lines between enums

Reply:
I think they're planning to add comments there

Reply:
Yes. However, I cannot afford to add comments.

Reply:
@0x5bfa can you remove the empty lines for now? We can add them back once comments are added.

Reply:
Added comments instead.